README.md

```diff
@@ -107,7 +107,8 @@ setup scripts:
 
 ```console
 ./setup_miniconda.sh
-./setup_env.sh -n probtest -u
+source miniconda/bin/activate
```

Comment:
I think this is better, but we could also just do `./setup_miniconda.sh -u`, right?

Reply:
Yes, you are right that would also work. This is just my prefered way of doing it :)
But we can mention both so everyone can choose for themself.